cmd/utils/flags.go

```diff
@@ -1957,14 +1958,17 @@ func MakeGenesis(ctx *cli.Context) *core.Genesis {
 
 // MakeChain creates a chain manager from set command line flags.
 func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chainDb ethdb.Database) {
+	return MakeChainWithEngine(ctx, stack, mockEngine.NewFaker())
```

Comment:
is this change necessary? I don't see this being used

Reply:
seems like a remainder from a previous attempt. removing it